gui/tools/autorepl.py

```diff
@@ -412,9 +413,9 @@ def write_results():
 
     # Grab map from remote system
     if recursive:
-        rzfscmd = '"zfs list -H -t snapshot -p -o name,creation -r \'%s\'"' % (remotefs_final)
+        rzfscmd = "'zfs list -H -t snapshot -p -o name,creation -r \"%s\"'" % (remotefs_final)
```

Comment:
What do you want to change by swapping ' and " here?

Reply:
Being consistent by using single-quotes and escaping double-quotes with other parts of the code.

Reply:
So it changes nothing, then we should avoid it.

Reply:
Understood. Will revert this.